service/src/java/org/apache/hive/service/cli/HiveSQLException.java

```diff
@@ -70,6 +72,11 @@ public HiveSQLException(Throwable cause) {
     super(cause);
   }
 
+  public HiveSQLException(Throwable cause, String queryId) {
```

Comment:
We should even deprecate the old constructors without queryId & remove in the subsequent release